app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,24 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int) -> None:
+
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return self._hummer_level + 4
+
+    def player_info(self) -> str:
+        return (
+            f"Dwarf warrior {self.nickname}. "
+            f"{self.nickname} has a hummer "
+            f"of the {self._hummer_level} "
+            f"level"
+        )
+
+    def eat_favourite_dish(self) -> None:
```

Comment:
The `eat_favourite_dish` method is currently defined to return `None`. However, the previous review mentioned it should return a `str`. Please verify the task requirements and adjust the return type if necessary.